packages/salesforcedx-vscode-core/package.json

```diff
@@ -75,12 +75,28 @@
   ],
   "main": "./out/src",
   "contributes": {
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "metadata",
+          "title": "Org Browser",
+          "icon": "resources/dep.svg"
+        }
+      ]
+    },
     "views": {
       "explorer": [
         {
           "id": "sfdx.force.tasks.view",
           "name": "%running_tasks_title_text%"
         }
+      ],
+      "metadata": [
+        {
+          "id": "metadata",
+          "name": "Metadata",
+          "when": "sfdx:display_tree_view"
```

Comment:
I think you mentioned this was temporary. Later it'll probably be something like `sfdx:project_opened && sfdx:default_username_has_no_change_tracking`